src/site/content/ko/blog/defining-core-web-vitals-thresholds/index.md

```diff
@@ -0,0 +1,232 @@
+---
+title: Core Web Vitals 메트릭 및 임계값 정의
+subhead: Core Web Vitals 임계값에 대한 연구 및 방법론
+authors:
+  - bmcquade
+description: Core Web Vitals 임계값에 대한 연구 및 방법론
+date: 2020-05-21
+updated: 2020-05-27
+hero: image/admin/WNrgCVjmp8Gyc8EbZ9Jv.png
+alt: Core Web Vitals 임계값에 대한 연구 및 방법론
+tags:
+  - blog
+  - performance
+  - web-vitals
+---
+
+[Core Web Vital](/vitals/#core-web-vitals)은 웹에서 실제 사용자 경험의 중요한 측면을 측정하는 필드 메트릭 세트입니다. Core Web Vitals에는 메트릭과 각 메트릭에 대한 대상 임계값이 포함되어 있어 개발자가 사이트 경험 품질이 "양호(good)", "개선 필요(needs improvement)" 또는 "나쁨(poor)" 중 어디에 속하는지 이해하는 데 도움이 됩니다. 이 게시물에서는 일반적으로 Core Web Vitals 메트릭에 대한 임계값을 선택하는 데 사용되는 접근 방식과 특정 Core Web Vitals 메트릭에 대한 임계값이 어떻게 선택되었는지 설명합니다.
+
+## 다시 짚어보기: Core Web Vitals 메트릭 및 임계값
+
+2020년 Core Web Vitals은 Large Contentful Paint(최대 콘텐츠풀 페인트, LCP), First Input Delay(최초 입력 지연, FID), Cumulative Layout Shift(누적 레이아웃 이동, CLS) 메트릭입니다. 각 메트릭은 사용자 경험의 다른 측면을 측정합니다. LCP는 인지된 로드 속도를 측정하고 페이지 로드 타임라인에서 페이지의 메인 콘텐츠가 로드된 지점을 표시합니다. FID는 응답성을 측정하고 사용자가 페이지와 처음 상호 작용을 시도할 때 느끼는 경험을 수량화합니다. CLS는 시각적 안정성을 측정하고 시각적 페이지 콘텐츠의 예기치 못한 레이아웃 이동을 수치화하여 표현합니다.
+
+각 Core Web Vitals 메트릭에는 성능을 "양호(good)", "개선 필요(needs improvement)" 또는 "나쁨(poor)"으로 분류하는 관련 임계값이 있습니다.
+
+<div class="w-stack w-stack--center w-stack--md">{% Img src="image/tcFciHGuF3MxnTr1y5ue01OGLBn2/ZZU8Z7TMKXmzZT2mCjJU.svg", alt="최대 콘텐츠풀 페인트 권장 임계값", width="400", height="350" %} {% Img src="image/tcFciHGuF3MxnTr1y5ue01OGLBn2/iHYrrXKe4QRcb2uu8eV8.svg", alt="최초 입력 지연 권장 임계값", width="400", height="350" %} {% Img src="image/tcFciHGuF3MxnTr1y5ue01OGLBn2/dgpDFckbHwwOKdIGDa3N.svg", alt="누적 레이아웃 이동 권장 임계값", width="400", height="350" %}</div>
```

Comment:
These images are duplicated, I think. compare to: https://raw.githubusercontent.com/GoogleChrome/web.dev/main/src/site/content/en/blog/defining-core-web-vitals-thresholds/index.md